pkg/objectstore/objectstore.go

```diff
@@ -145,15 +145,20 @@ func s3Config(ctx context.Context, config ProviderConfig, secret *Secret) (stowK
 	return stows3.Kind, cm, nil
 }
 
-func gcsConfig(ctx context.Context, secret *Secret) (stowKind string, stowConfig stow.Config, err error) {
+func gcsConfig(ctx context.Context, config ProviderConfig, secret *Secret) (stowKind string, stowConfig stow.Config, err error) {
 	var configJSON string
 	var projectID string
+	cm := stow.ConfigMap{}
 	if secret != nil {
 		if secret.Type != SecretTypeGcpServiceAccountKey {
 			return "", nil, errors.Errorf("invalid secret type %s", secret.Type)
 		}
 		configJSON = secret.Gcp.ServiceKey
 		projectID = secret.Gcp.ProjectID
+		if config.Region != "" {
+			cm[stowgcs.ConfigLocation] = config.Region
+			cm[stowgcs.ConfigStorageClass] = REGIONAL
```

Comment:
Alternatively `config.StorageClass` could be added but it would only be relevant for GCS